applications/io_board/targets/freertos.armv7m.z-dev-board-123/HwInit.cxx

```diff
@@ -1 +0,0 @@
-../../../../boards/ti-ek-tm4c123gxl-z-dev-board/HwInit.cxx
```

Comment:
I noticed that too. I didn't touch or remove any of the other boards files, I checked out the upstream/pic32mx-ram-optimize branch, Perhaps that branch isn't complete?